### PR TITLE
Add docker-compose config

### PR DESCRIPTION
**Pull request by andrewdotn**

### CODE REVIEW

File: CreeDictionary/CreeDictionary/settings.py [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "API.apps.APIConfig",
     "CreeDictionary.apps.CreeDictionaryConfig",
     "morphodict.apps.MorphodictConfig",
+    "DatabaseManager",
```
Comment (eddieantonio) [inline CreeDictionary/CreeDictionary/settings.py:62]: I love splitting this up into apps!

File: CreeDictionary/setup.py [hunk 2]
```diff
@@ -1,11 +0,0 @@
-from setuptools import find_packages, setup
```
Comment (eddieantonio) [inline CreeDictionary/setup.py:1]: THANK.
YOU.

File: docker/Dockerfile [hunk 3]
```diff
@@ -46,25 +50,39 @@ RUN set -ex \
...
 
-# Setup Python deps
-ADD Pipfile /app/Pipfile
-ADD Pipfile.lock /app/Pipfile.lock
-
 WORKDIR /app/
 
+# Setup Python deps
+ADD --chown=${WSGI_USER} Pipfile .
+ADD --chown=${WSGI_USER} Pipfile.lock .
```
Comment (eddieantonio) [inline docker/Dockerfile:57]: Ooh! Much cleaner than doing this in a `RUN` command!

File: docker/Dockerfile [hunk 4]
```diff
@@ -46,25 +50,39 @@ RUN set -ex \
...
+
 RUN PIPENV_VENV_IN_PROJECT=1 pipenv install --deploy
 
+ADD --chown=${WSGI_USER} package.json .
+ADD --chown=${WSGI_USER} package-lock.json .
+# cypress is a big, heavy package that takes a long time to install and
+# that we absolutely don’t need in production
+RUN grep -Ev '"@?cypress' package.json | sponge package.json
```
Comment (eddieantonio) [inline docker/Dockerfile:65]: Excellent!

File: docker/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Docker for itwêwina
+
+The current setup was created with the intent of using it in production,
+not for development.
+
+  - Run `sqlite3 db.sqlite3` to create an empty database file
+
+    Otherwise `docker-compose` will create a *directory* instead
```
Comment (eddieantonio) [inline docker/README.md:8]: wat

File: docker/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+  - Populate the test dictionary with
+
+        ./docker-django-manage.py xmlimport import \
+            ../CreeDictionary/res/test_dictionaries/crkeng.xml
+
+## Future work
+
+  - smaller docker image without dev dependencies through multi-stage
+    builds
```
Comment (eddieantonio) [inline docker/README.md:27]: 👍🏼

**Review by eddieantonio — APPROVED** [review]
This is some great stuff! 👍🏼

File: docker/README.md (hunk 5 above)
Comment (andrewdotn) [reply docker/README.md:8]: When docker is mounting host paths, those paths can be either directories or files. Docker will create the path if it does not exist. Creating an empty directory makes a bit more sense than creating a zero-byte file. If docker were being designed anew, a trailing slash could be required to indicate a directory, but there are already a ton of things out there assuming that specifying a non-existent path without a trailing slash will result in directory creation.

**Pull request merged**